pkg/volume/lvmlocalpv.go

```diff
@@ -57,7 +57,7 @@ func GetLVMLocalPV(c *client.K8sClient, pvList *corev1.PersistentVolumeList, ope
 			attachedNode = lvmVol.Spec.OwnerNodeID
 			rows = append(rows, metav1.TableRow{
 				Cells: []interface{}{
-					ns, pv.Name, customStatus, version, pv.Spec.Capacity.Storage(), pv.Spec.StorageClassName, pv.Status.Phase,
+					ns, pv.Name, customStatus, version, pv.Spec.Capacity.Storage().String(), pv.Spec.StorageClassName, pv.Status.Phase,
```

Comment:
This has been added to make sure tests pass, as `4Gi` resource.Quantity was not equal to `4Gi` resource.Quantity.

Another fun debugging session helped me use typed variables for PVBound, ReadWriteOnce.

Reply:
Please humaize this using, util.ConvertToIBytes(), it accepts any unit and converts to IBytes,

Reply:
> Please humaize this using, util.ConvertToIBytes(), it accepts any unit and converts to IBytes,

It's not required as I'm getting them from the `PV`, which is a corev1 kubernetes resource, that's managed automatically as it's not a string but a [metav1.resource.Quantity](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity). The only problem with this one is that it doesn't support representing the size in XYZ(i)**bits** ∀ `{K, M, G, T...}`. Humanizing is required if the sizes are typed as `string`.